src/nvme/mi.c

```diff
@@ -700,7 +696,17 @@ int nvme_mi_admin_admin_passthru(nvme_mi_ctrl_t ctrl, __u8 opcode, __u8 flags,
 		resp.data_len = data_len;
 	}
 
+	if (timeout_ms != 0) {
+		/* When user forgets to set timeout_ms, the timeout_ms will be 0.
```

Comment:
The logic here is sound, but you're documenting the timeout == 0 case in the condition where the timeout is not zero.

Perhaps something like:

```
    /* if the user has specified a custom timeout, save the current timeout and override */
```

\- it's not necessarily an error ("user forgets") to pass zero here.

Reply:
Done.